release/nightly_tests/nightly_tests.yaml

```diff
@@ -366,7 +366,6 @@
     timeout: 3600
     prepare: python wait_cluster.py 2 600
     script: python placement_group_tests/long_running_performance_test.py --num-stages 2000
-  stable: false
```

Comment:
Unrelated to this PR< but this test has passed consistently for a long time, so I'd like to remove this flag.